engine-tests/src/tests/pause_contract.rs

```diff
@@ -0,0 +1,69 @@
+use crate::test_utils;
+
+#[test]
+fn test_pause_contract_require_owner() {
+    let mut runner = test_utils::deploy_evm();
+    let aurora_account_id = runner.aurora_account_id.clone();
```

Comment:
Clone is redundant here because you need a reference
```suggestion
    let aurora_account_id = &runner.aurora_account_id;
```

Reply:
I tried but the borrow checker is complaining about it.

let aurora_account_id = &runner.aurora_account_id;
  |                             ------------------------- immutable borrow occurs here
7 |
8 |     let result = runner.call("pause_contract", aurora_account_id, vec![]);
  |                  ^^^^^^^----^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |                  |      |
  |                  |      immutable borrow later used by call
  |                  mutable borrow occurs here

Reply:
Got it.